include/aikido/constraint/uniform/RnConstantSampler.hpp

```diff
@@ -26,14 +28,23 @@ class RnConstantSampler : public constraint::Sampleable
     createSampleGenerator() const override;
 
   /// Returns constant value to be generated by this sampler.
-  const Eigen::VectorXd& getConstantValue() const;
+  const Vectord& getConstantValue() const;
 
 private:
-  std::shared_ptr<statespace::Rn> mSpace;
-  Eigen::VectorXd mValue;
+  std::shared_ptr<statespace::R<N>> mSpace;
+  Vectord mValue;
 };
 
+using R0ConstantSampler = RnConstantSampler<0>;
+using R1ConstantSampler = RnConstantSampler<1>;
+using R2ConstantSampler = RnConstantSampler<2>;
+using R3ConstantSampler = RnConstantSampler<3>;
+using R6ConstantSampler = RnConstantSampler<6>;
+using RxConstantSampler = RnConstantSampler<Eigen::Dynamic>;
```

Comment:
Which means, we also rename the template class `RnConstantSampler` to `RConstantSampler`, right?